vault/logical_system_helpers.go

```diff
@@ -92,6 +92,10 @@ var (
 	}
 
 	checkRaw = func(b *SystemBackend, path string) error { return nil }
+
+	wrapHandleRaftRemovePeer = func(b *SystemBackend) framework.OperationFunc {
```

Comment:
@alexanderbez I think this is causing oss to ent merge conflict. You may want to fix the signature here.